library/ecp_curves.c

```diff
@@ -1195,6 +1240,76 @@ static int ecp_mod_p255( mbedtls_mpi *N )
 }
 #endif /* MBEDTLS_ECP_DP_CURVE25519_ENABLED */
 
+#if defined(MBEDTLS_ECP_DP_CURVE448_ENABLED)
+
+/* Size of p448 in terms of mbedtls_mpi_uint */
+#define P448_WIDTH      ( 448 / 8 / sizeof( mbedtls_mpi_uint ) )
+
+/* Number of limbs fully occupied by 2^224 (max), and limbs used by it (min) */
+#define DIV_ROUND_UP( X, Y ) ( ( X + Y - 1 ) / ( Y ) )
```

Comment:
Please add brackets around all uses of `X` and `Y`; e.g. using `<<` within `X` or `Y` could currently lead to wrong results.